lib/validation.js

```diff
@@ -37,19 +37,14 @@ function build (context, compile, schemas) {
     // The header keys are case insensitive
     //  https://tools.ietf.org/html/rfc2616#section-4.2
     const headersSchemaLowerCase = {}
-    for (const k in headers) {
-      if (headers.hasOwnProperty(k) !== true) continue
-      headersSchemaLowerCase[k] = headers[k]
-    }
+    Object.keys(headers).forEach(k => headersSchemaLowerCase[k] = headers[k])
```

Comment:
Increasing test coverage should _not_ include changing the code being tested. This sort of thing should be addressed in another PR.

Reply:
Fair point! Would you like me to revert this change and make it in a separate PR?

Reply:
That would be most appropriate.

Reply:
Okay removed commit; going to add it to another PR now